server/machines/machines.repository.go

```diff
@@ -0,0 +1,60 @@
+package machines
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+type MachinesRepository struct {
+	Db *sql.DB
+}
+
+func (mr *MachinesRepository) GetAll() ([]Machine, error) {
+	rows, err = r.DB.Query("SELECT id, name, cpu_count, total_disk_space FROM machines")
```

Comment:
you wrote 'r.DB.Query', but should 'mr.DB.Query'

Reply:
yep, thanks